.teamcity/src/main/kotlin/common/extensions.kt

```diff
@@ -212,7 +212,7 @@ fun BuildType.killProcessStep(stepName: String, daemon: Boolean) {
             executionMode = BuildStep.ExecutionMode.ALWAYS
             tasks = "killExistingProcessesStartedByGradle"
             gradleParams =
-                buildToolGradleParameters(daemon).joinToString(separator = " ")
+                (buildToolGradleParameters(daemon) + "-Dscan.tag.CleanUpBuild").joinToString(separator = " ")
```

Comment:
We have a `buildScanTag()` method for doing this.

Reply:
Thanks! Good to know!